public/src/components/login.jsx

```diff
@@ -17,19 +17,18 @@ class LogInForm extends Component {
   }
 
   renderField(field) {
-    const className = `form-group ${field.meta.touched && field.meta.error ? 'has-danger' : ''}`;
+    const className = `form-group ${field.meta.touched && field.meta.error ? 'has-error' : ''}`;
 
     return (
       <div className={className}>
         <label>{field.label}</label>
         <input
-          // type={!!field.name && field.name === 'password' ? 'password' : 'text'}
           className="form-control"
           placeholder={field.label}
           type={field.label}
           {...field.input}
         />
-        <div className="text-help has-danger">
+        <div className="text-help has-error">
```

Comment:
Is there a reason the class is being used again here? I'm wondering if it should just be text-danger